src/test/unit/parameter_group.ld

```diff
@@ -0,0 +1,10 @@
+SECTIONS {
+  .pg_registry :   SUBALIGN(4)  /* force 4-byte alignment for pg_regisry. Gcc default is 32 bytes, causing padding insertion and breaking the init structure. */
```

Comment:
ld documentation reachable through google is quite outdated :-(, did take some time to find this